R/post-body.R

```diff
@@ -24,3 +34,33 @@ parseBody <- function(body){
   }
   ret
 }
+
+
+#' @importFrom utils URLdecode
+#' @noRd
+parseMultipart <- function(body, boundary){
+  function(val, req, res, errorHandler){
+    tryCatch({
+      if (!requireNamespace("webutils", quietly = TRUE)) {
+        stop("The webutils package is not available but is required in order
+          to use the functionality to POST binary files",
+             call. = FALSE)
+      }
+      # Is there data in the request?
+      if (is.null(body) || length(body) == 0 || body == "") {
+        return(list())
+      }
+      parsed_binary <- webutils::parse_multipart(body, boundary)
+      file_name <- parsed_binary$myfile$filename
```

Comment:
I'm having a hard time finding where `myfile` comes from? Is that baked in to `parse_multipart`'s return value?